slides/3-foundations-of-javascript.html

```diff
@@ -476,6 +595,14 @@ <h2><i class="fab fa-js"></i> Foundations of JavaScript</h2>
                             a **= 1; // exponentiation
                             ```
 
+                            JavaScript also has increment and decrement unary operators
```

Comment:
Added content : increment operators will be a question in the next week's slides, it should be mentionned somewhere.